src/api/handlers/guild.ts

```diff
@@ -769,6 +769,16 @@ export function getGuild(guildID: string, counts = true) {
 }
 
 /** Returns the guild template if it exists */
+export function getTemplate(templateCode: string) {
+  return RequestManager.get(endpoints.GUILD_TEMPLATE(templateCode)) as Promise<
+    GuildTemplate
+  >;
+}
+
+/** 
+ * Returns the guild template if it exists 
+ * @deprecated This function no longer works
```

Comment:
Maybe a better deprecation note?

Reply:
I think I will just redirect to use the `getTemplate` function this function can then be removed in v11.

Reply:
No, that's not ideal.

Reply:
Just remove it from entirely in a new PR, which would be a breaking change.

Reply:
Well what do you want then?